tests/console/docker.pm

```diff
@@ -100,12 +100,18 @@ sub run {
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
     # containers state can be saved to a docker image
-    my $output = script_output("docker container exec $container_name zypper -n in curl", 300);
-    die('error: curl not installed in the container') unless ($output =~ m/Installing: curl.*done/);
+    # note that here script_run do not return in case of failure,
+    # so returned value is more an exit_code than an output string.
+    my $exit_code = script_run("docker container exec $container_name zypper -n in curl", 300);
+    if ($exit_code) {
+        record_soft_failure('poo#40958 - curl install failure, try with force-resolution.');
```

Comment:
Please only reference product bugs with "soft failure". I think the workaround is a good idea though. How about you just replace `record_soft_failure` with `record_info`? Just like `record_info 'poo#40958', '…';`

Reply:
use record_info as suggested